gems/aws-sdk-core/spec/aws/plugins/request_compression_spec.rb

```diff
@@ -121,13 +121,6 @@ def expect_uncompressed_body(resp, body)
       end
 
       context 'request compression operation' do
-        it 'sets user-agent metric for the operation' do
```

Comment:
Are we opting not to write unit tests for user-agent metrics in general? Or will be there one spec that covers all current + newly added metric values?

Reply:
I don't think it makes sense to test these. The feature is generally tested already in the user agent spec IIRC.